### ISSUE TITLE
x87: wrong parameter for fyl2x

### ISSUE THREAD

**Issue by rfalke** [uxmal/reko#972]
Subject: https://github.com/rfalke/decompiler-subjects/blob/master/from_holdec/i386_x87_fpu/ia32_elf
Output:
```
void FYL2X()
{
	real64 rLoc1_67 = lg2(1.0) * 1.0;
	printf("FYL2X(para, base=2):  should=                             is=%12f/%15e\n", rLoc1_67, rLoc1_67);
	real64 rLoc1_73 = lg2(1.0) * 1.0;
	printf("FYL2X(1000, base=2):  should=    9.965784/   9.965784e+00 is=%12f/%15e\n", rLoc1_73, rLoc1_73);
	real64 rLoc1_79 = lg2(0.693147180559945) * 0.693147180559945;   <==================
	printf("FYL2X(1000, base=e):  should=    6.907755/   6.907755e+00 is=%12f/%15e\n", rLoc1_79, rLoc1_79);
	real64 rLoc1_85 = lg2(0.301029995663981) * 0.301029995663981;  <==================
	printf("FYL2X(1000, base=10): should=    3.000000/   3.000000e+00 is=%12f/%15e\n", rLoc1_85, rLoc1_85);
}
```

fyl2x takes two args st0 and st1. reko only uses one.

**Comment by rfalke** [comment]
It is correct for fyl2xp1.

**Comment by uxmal** [comment]
Fixed in aec4f59
